coverage_config_x86_64.json

```diff
@@ -1,5 +1,5 @@
 {
-  "coverage_score": 77.28,
+  "coverage_score": 76.49,
   "exclude_path": "",
```

Comment:
Should we exclude the template crate from coverage?

Reply:
No. I think it is better to have enough tests to cover the template crate too.. and include it in coverage.

Reply:
I tend to agree. We want the template to be the goto "reference" for new devices so it should be as clean and idiomatic as possible which includes having good coverage.